ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -913,6 +913,9 @@ runThreadNetwork systemTime ThreadNetworkArgs
                   bfcMaxConcurrencyBulkSync = 1
                 , bfcMaxConcurrencyDeadline = 2
                 , bfcMaxRequestsInflight    = 10
+                , bfcDecisionLoopInterval   = 0.0 -- Mock testsuite can use sub-second slot
```

Comment:
We decided to set it to `0` for now, which disables the rate limiter during the ThreadNet tests.

If this interval is greater than the slot length, then this will lead to test failures. In this case, some BF thread would eventually sleep for a duration that spans the next slot onset, but the tests currently assume that the nodes are more responsive than that -- roughly: "all nodes are fully synced by the end of every slot".

See https://input-output-rnd.slack.com/archives/CFKLUH4R0/p1595255785454400 for the discussion.